_data/developer.yml

```
@@ -563,6 +563,7 @@ websites:
       img: unity.png
       tfa: Yes
       sms: Yes
+      software: Yes
       doc: https://support.unity3d.com/hc/en-us/articles/208764976-How-do-I-enable-Two-Factor-Authentication-on-my-account-
```

No need for the documentation URL to be so long 😄 
```suggestion
      doc: https://support.unity3d.com/hc/en-us/articles/208764976
```